repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
@ajs6f: AFAICT, no URIs defined in the Fedora ontology are dereferenced at any point in the code. In fact, most of the code consists merely of string concatenation (namespace + property name like so: `REPOSITORY_NAMESPACE + "Binary"`). Which is to say, the namespace document could contain entirely different values and the runtime code wouldn't care. I see this as a problem, which is why I created the `fcrepo-vocabulary` project, which at least verifies that any fields defined in the JAVA class are at least present in the ontology.